pkg/component/nodemanagement/nodeproblemdetector/node_problem_detector.go

```diff
@@ -308,6 +309,29 @@ func (c *nodeProblemDetector) computeResourcesData() (map[string][]byte, error)
 			},
 		}
 
+		service = &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      serviceName,
+				Namespace: metav1.NamespaceSystem,
+				Labels: map[string]string{
+					"app.kubernetes.io/instance": "shoot-core",
+					"app.kubernetes.io/name":     labelValue,
+					v1beta1constants.LabelApp:    labelValue,
+					v1beta1constants.GardenRole:  v1beta1constants.GardenRoleSystemComponent,
+				},
```

Comment:
You can use `getLables()` here also.
```suggestion
				Labels: utils.MergeStringMaps(getLabels(), map[string]string{
					v1beta1constants.LabelApp:   labelValue,
					v1beta1constants.GardenRole: v1beta1constants.GardenRoleSystemComponent,
				}),
```

Reply:
Thank you, @acumino! I applied your suggestion to similar occurrences in the same file as well.